third-party/CMakeLists.txt

```diff
@@ -198,6 +199,9 @@ set(LIBICONV_SHA256 ccf536620a45458d26ba83887a983b96827001e92a13847b45e4925cc891
 set(TREESITTER_C_URL https://github.com/tree-sitter/tree-sitter-c/archive/6002fcd.tar.gz)
 set(TREESITTER_C_SHA256 46f8d44fa886d9ddb92571bb6fa8b175992c8758eca749cb1217464e512b6e97)
 
+set(TREESITTER_URL https://github.com/tree-sitter/tree-sitter/archive/0.17.1.zip)
```

Comment:
Have you checked whether the runtime update resolve the issue with newest Python/Go/Java parsers (they are locked to the revisions before the 0.17.1 CLI update)? Can parsers generated with old tree-sitter CLI (0.16) still be used without problems on the new runtime?

Reply:
I did not tried, but I guess it would be okay to use older generated parsers. This needs testing.